src/Assignment6/Shape_Hierarchy/Shape.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Shape_Hierarchy
+{
+    /// <summary>
+    /// Shape abstract class
+    /// </summary>
+    public abstract class Shape
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Shape"/> class.
+        /// It sets the colour, Input1 and Input2 of the dimensions of the shapes
+        /// </summary>
+        /// <param name="colour">It takes the colour as a string</param>
```

Comment:
```suggestion
        /// <param name="colour">Color of the shape.</param>
```

Update all the param and method descriptions similar to this.

Reply:
I have resolved it.